src/app/notifications-page/notifications-list/notifications-list.component.ts

```diff
@@ -219,12 +219,27 @@ export class NotificationsListComponent implements OnInit {
       const isDiamond = cctMeta.DiamondLevel > 0;
       result.icon = isDiamond ? "icon-diamond fc-blue" : "fas fa-paper-plane fc-blue";
       const diamondText = isDiamond
```

Comment:
Can we make what one giant if else block for `isDiamond { result.action = '' } else { result.action = '' }` now that there appears to be no overlap between these two?